### PR TITLE
Server setup

### PR DESCRIPTION
**Pull request by JDziewonski98**
# Summary

Set up boilerplate / basic infrastructure. Can connect to a postgres container and CRUD it.

## Test Plan

todo :/ 

## Related Issues

none

### CODE REVIEW

File: backend/src/config/routes.ts
Comment (tedspare) [inline backend/src/config/routes.ts:7]: Does app have a type?

**Review by tedspare — APPROVED** [review]
This looks good to me! I added one minor comment but couldn't find any meaningful room for feedback. Really thoroughly written!

What about adding a backend README with a summary (1-2 sentences) of major design decisions for team members who are new to this stack? For example, "The MuralController maps endpoints and methods to database operations"

I'll add another team member who is more experienced in backend before merging.

**Review by tedspare — APPROVED** [review]
LGTM

**Comment by Tony9984** [comment]
if u told me u were new to typescript i wouldn't have believed you haha, I'll merge it if everyone's fine with it?

**Review by Tony9984 — APPROVED** [review]

**Pull request merged**
